web/src/components/tasks/business-formation/name/NexusSearchBusinessNameStep.test.tsx

```diff
@@ -52,34 +61,55 @@ function setupMockAnalytics(): typeof analytics {
 }
 const mockAnalytics = analytics as jest.Mocked<typeof analytics>;
 
-describe("<NexusSearchBusinessNameStep />", () => {
-  const initialUserData = generateUserData({
-    formationData: generateFormationData({ formationFormData: createEmptyFormationFormData() }),
-    profileData: generateProfileData({
-      businessName: "",
-      nexusDbaName: "",
-      needsNexusDbaName: false,
-    }),
-  });
+describe("Formation - NexusSearchBusinessNameStep", () => {
```

Comment:
This is essentially a rewrite of the file, but all of the tests should be the same.